pkg/api/preferences.go

```diff
@@ -16,6 +16,22 @@ const (
 	lightTheme   string = "light"
 )
 
+// PUT /api/preferences/set-json-data
+func (hs *HTTPServer) SetJsonData(c *models.ReqContext) response.Response {
```

Comment:
Because this is a global function name within the api I'd give it a slightly more descriptive name, the same is true for the query, command and sqlstore function names.
```suggestion
func (hs *HTTPServer) SetPreferencesJsonData(c *models.ReqContext) response.Response {
```